tests/test_api.py

```diff
@@ -145,17 +189,24 @@ def assert_test_project(self, project, name=None):
         assert project['name'] == self._project_name if name is None else name
         assert isinstance(project['id'], int)
 
-    def assert_test_project_result(self, result, name=None):
+    def assert_test_project_result(self, result, name=None, logs_limit=None):
+        print(result)
         assert len(result) == 9
         assert isinstance(result['commands'], list)
-        assert isinstance(result['logs'], list)
+        self.assert_test_logs(result['logs'], logs_limit)
         assert isinstance(result['args'], list)
         assert isinstance(result['snapshots'], list)
         assert isinstance(result['isUnregistered'], bool)
         assert isinstance(result['id'], int)
         assert result['pathName'].startswith(self._project_path)
         assert result['name'] == name
 
+    def assert_test_logs(self, logs, logs_limit=None):
+        assert isinstance(logs, list)
+        assert logs_limit is None or logs_limit == -1 \
+            or len(logs) == min(7, logs_limit)
+        print(len(logs), logs_limit)
```

Comment:
remove the print debug, or if this code has a point, please add comment to complement.